src/pretix/base/services/orders.py

```diff
@@ -546,7 +605,7 @@ def _reissue_invoice(self):
             generate_invoice(self.order)
 
     def _check_complete_cancel(self):
-        cancels = len([o for o in self._operations if isinstance(o, self.CancelOperation)])
+        cancels = len([o for o in self._operations if isinstance(o, self.CancelOperation) or isinstance(o, self.SplitOperation)])
```

Comment:
I believe rewriting this as `isinstance(o, (self.CancelOperation, self.SplitOperation))` is more pythonic 🙂